frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -317,7 +328,7 @@ export default {
     z-index: 30;
 
     .resList {
-      max-height: 22rem;
+      max-height: 33rem;
```

Comment:
Can we even make the width larger? For desktop users, the current width of the GemSearch window is usually far away from the width of the browser. For mobile user, its the same.

Reply:
Resolved in commit 809b9df34a4e1e30b1bc5e3292835a7f43661afc